fail2ban/server/filter.py

```diff
@@ -898,22 +898,22 @@ def textToIp(text, useDns):
 		return ipList
 
 	@staticmethod
-	def cidr(i, n):
-		""" Convert an IP address string with a CIDR mask into a 32-bit
-			integer.
+	def netAddr2Dec(ipstring, cidr):
+		""" Convert a string IPv4 address with CIDR
+		to decimal IP address of first address in block (network address).
 		"""
 		# 32-bit IPv4 address mask
 		MASK = 0xFFFFFFFFL
-		return ~(MASK >> n) & MASK & DNSUtils.addr2bin(i)
+		return ~(MASK >> cidr) & MASK & DNSUtils.addr2dec(ipstring)
 
 	@staticmethod
-	def addr2bin(string):
-		""" Convert a string IPv4 address into an unsigned integer.
+	def addr2dec(ipstring):
```

Comment:
As the author of IPCalcEx (Android App), I may be biased (my app uses binary string and decimal as I portrayed them here). With regards to fail2ban, that argument isn't as relevant regarding a function's name - so I will cordially disagree, since it is trivial in comparison to the argument it would require to even potentially persuade you :) I see your point, and I see mine :)

For the record, I still think decimal is a more accurate description to the value returned :)